src/sagas/featureToggle.js

```diff
@@ -144,25 +143,19 @@ export function* monitorFeatureFlags(currentRetry = 0) {
 
 export function* setupUnleashListeners(unleashClient) {
   const channel = eventChannel((emitter) => {
-    const listener = (state) => emitter(state);
-
-    unleashClient.on(UnleashEvents.UPDATE, () => emitter({
-      type: 'FEATURE_TOGGLE_UPDATE',
-    }));
-
-    unleashClient.on(UnleashEvents.READY, () => emitter({
-      type: 'FEATURE_TOGGLE_READY',
-    }));
+    const l1 = () => emitter({ type: 'FEATURE_TOGGLE_UPDATE' });
+    const l2 = () => emitter({ type: 'FEATURE_TOGGLE_READY' });
+    const l3 = (err) => emitter({ type: 'FEATURE_TOGGLE_ERROR', data: err });
```

Comment:
🚲 It doesn't need to change, but it would be better to have a proper name for the listeners, like `update_listener`, `ready_listener`, `error_listener`. This would be nice to have while reading an error stack.